repo-scripts/size-analysis/package.json

```diff
@@ -16,18 +18,22 @@
     "type-check": "tsc -p . --noEmit"
   },
   "dependencies": {
-    "typescript": "3.9.7",
+    "@firebase/app": "0.6.10",
+    "@firebase/util": "0.3.1",
```

Comment:
These are used in tests only, right? If so, they should be in devDependencies.

Reply:
`deepCopy` from util is used in analysis-helper.ts. but i will move app to `devDependencies`

Reply:
SG